compiler/qsc_parse/src/lib.rs

```diff
@@ -51,6 +51,9 @@ enum ErrorKind {
     #[error("expected {0}, found {1}")]
     #[diagnostic(code("Qsc.Parse.Token"))]
     Token(TokenKind, TokenKind, #[label] Span),
+    #[error("expected item after doc comment")]
```

Comment:
I'm on the fence regarding the error text. While "doc comment" is a very common term of art, I'm tempted to say the abbreviate should be skipped to help readability. However, "documentation comment" looks super weird to my eyes. Perhaps it's fine to leave as is and see if we get any other feedback on it?

Reply:
Maybe you'd prefer "docstring"?

Reply:
I think leaving it as-is should be fine, I'm probably overthinking it!